spec/game_logic_spec.rb

```diff
@@ -0,0 +1,160 @@
+#spec/calculator_spec.rb
+require './lib/game_logic'
```

Comment:
Please use `require_relative` to link files in your directory. 
Using `require_relative` for files in your directory instead of `require` is a ruby convention.